### PR TITLE
Sonarqube integration

### PR DESCRIPTION
**Pull request by Sri-hari**

### CODE REVIEW

**Comment by AmitRoushan** [comment]
@Sri-hari Please add description for the MR

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
+          SONAR_TOKEN: a5b664c1d0bbd7953597b12ab1a9126aede89867
```
Comment (sodagitops) [inline .github/workflows/main.yml:15]: Pls update token

Comment (sushanthakumar) [inline .github/workflows/main.yml:15]: Please update the hardcoded token

**Review by sushanthakumar — APPROVED** [review]

**Comment by sushanthakumar** [comment]
LGTM

**Pull request merged**